src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -7371,21 +7371,11 @@ definitions:
       properties:
         type: object
         additionalProperties:
-          type: string
+          type: object
       capabilities:
-        $ref: '#/definitions/Capabilities'
-  Capabilities:
```

Comment:
Are we changing this because capabilities are open ended?

Reply:
Changed to reflect what is here, not to limit it to those 3 values:
https://github.com/Alfresco/alfresco-remote-api/blob/master/src/main/java/org/alfresco/rest/api/model/Person.java#L73

Reply:
Nice, thanks for the clarification.